core/src/view/view.h

```diff
@@ -107,6 +107,9 @@ class View {
     /* Gets the normal matrix; transforms surface normals from model space to camera space */
     const glm::mat3& getNormalMatrix() const { return m_normalMatrix; }
 
+    /* Returns the eye position in world space */
+    const glm::vec3& getEye() const { return m_eye; }
```

Comment:
Being consistent with the rest of the view module, should we change everything in here then?